src/ir_Mitsubishi.cpp

```diff
@@ -553,11 +553,25 @@ void IRMitsubishiAC::setVane(const uint8_t position) {
   remote_state[9] |= pos;
 }
 
+// Set the requested wide-vane operation mode of the a/c unit.
+void IRMitsubishiAC::setWideVane(const uint8_t position) {
+  uint8_t pos = std::min(position, kMitsubishiAcWideVaneAuto);  // bounds check
+  pos &= 0b1111;
```

Comment:
[nitpick] As you are shifting by 4 bits in the next line. This is line is redundant, remove it. i.e. The upper 4 bits will be gone after the next line.

Reply:
Sorry I left that by mistake.